README.md

```diff
@@ -59,6 +57,9 @@ docker build -f Dockerfile.web -t ocaml-ci-web .
 The `stack.yml` contains the configuration used on the live system.
 You'll have to register your own GitHub app to be able to test the services locally.
 
+If you want it to update to changes in opam-repository automatically you'll also need
+to register a webhook there sending push events to the CI's `/webhooks/github` path.
```

Comment:
Fantastic! Does this webhook also need a secret, or is it just an open ping channel?

Reply:
It doesn't use a secret at the moment. It responds to any event by querying GitHub, so it's just a DoS if someone sends fake events.

Reply:
Sounds good.